frontend/src/app/features/team-planner/team-planner/planner/tp-assignee.sass

```diff
@@ -0,0 +1,30 @@
+.tp-assignee
+  display: flex
+  max-width: 100%
+
+  &--principal
+    max-width: 100%
+    min-width: 0 // See: https://css-tricks.com/flexbox-truncated-text/
+    flex-grow: 1
+    flex-shrink: 1
+    
+  &--remove
+    background: white
+    border-radius: 50%
+    margin: 0
+    padding: 0
+    padding-left: 0.5rem
+    flex-grow: 0
+    flex-shrink: 0
+    width: 0
+    border: 0
+    background: transparent
+    cursor: normal
+    pointer-events: none
+    opacity: 0
+
+  @at-root &--remove:focus, &:hover &--remove, #{$spec-active-selector} &--remove
```

Comment:
Could you give me an example of why this is necessary again? You mentioned that selenium does not work correctly with hover & focus, but I'm not aware of any issues there.

Reply:
For the browser to mark an element as currently focused or hovered, the browser window itself will have to be focused/hovered. You can test this in your normal browser; if you enter a text area it's focused but if you then go to a different window it wil not be focused anymore.

In an automated testing environment, the browser never has focus, so `:focus`, `:hover` and similar JS constructs do not work. We'd have to do some X11 hacking or similar.